src/pkgcore/config/basics.py

```diff
@@ -597,9 +440,9 @@ def convert_hybrid(central, value, arg_type: str):
 
 # "Invalid name" (pylint thinks these are module-level constants)
 # pylint: disable-msg=C0103
-HardCodedConfigSection = partial(FakeIncrementalDictConfigSection, convert_asis)
-ConfigSectionFromStringDict = partial(FakeIncrementalDictConfigSection, convert_string)
-AutoConfigSection = partial(FakeIncrementalDictConfigSection, convert_hybrid)
+HardCodedConfigSection = partial(DictConfigSection, convert_asis)
```

Comment:
later PR's should cleanup the type conversion logic.  If you trace through those, they're fairly fucking gross, but they were necessary shims (at the time) to make things like `pquery --add-config` work properly w/ the frameworks in use at the time.

They can be improved, is the TL;DR.